solr/test-framework/src/java/org/apache/solr/embedded/JettySolrRunner.java

```diff
@@ -283,6 +283,7 @@ private void init(int port) {
     QueuedThreadPool qtp = new QueuedThreadPool();
     qtp.setMaxThreads(THREAD_POOL_MAX_THREADS);
     qtp.setIdleTimeout(THREAD_POOL_MAX_IDLE_TIME_MS);
+    qtp.setIdleTimeoutDecay(THREAD_POOL_MAX_THREADS);
```

Comment:
Do we want a different constant for idleTimeoutDecay or should this actually be tied to maxThreads?

Reply:
For `JettySolrRunner` it probably doesn't matter. For clarity though I agree -- propose to make a different constant, same value?:
```java
  private static final int THREAD_POOL_MAX_THREADS = 10000;
  private static final int THREAD_POOL_DECAY_RATE = THREAD_POOL_MAX_THREADS;
```

Reply:
👍 on a new constant, same value. Provides better clarity.